### ISSUE TITLE
broadcast of token dragging as a player is not working anymore

### ISSUE THREAD

**Issue by DarKDinDoN** [wsaunders1014/ShowDragDistance#34]
Hello,

Seems that with the latest version, when a player is dragging his token, the path is not broadcasted to others.

**Comment by DarKDinDoN** [comment]
Yep, confirmed.

Here: https://github.com/wsaunders1014/ShowDragDistance/blob/master/showdragdistance.js#L525
Only the GM can broadcast the path.

Quick fix is:
```javascript
if(!this.data.hidden && ((game.user.isGM && altPressed) || !game.user.isGM)){
    [...]
}
```

**Comment by wsaunders1014** [comment]
Fixed in 2.1.3
